crates/matrix-sdk-crypto/src/gossiping/machine.rs

```diff
@@ -1032,18 +1037,20 @@ mod tests {
         ReadOnlyAccount::new(alice_id(), alice2_device_id())
     }
 
-    fn bob_machine() -> GossipMachine {
-        let user_id = Arc::from(bob_id());
-        let account = ReadOnlyAccount::new(&user_id, alice_device_id());
+    fn bob_machine(user_id: &UserId) -> GossipMachine {
```

Comment:
Okay, now this I'm pretty sure this function name no longer makes sense 😅 

Maybe `test_gossip_machine()` or sth. like that?

Reply:
Right, there we go https://github.com/matrix-org/matrix-rust-sdk/pull/901/commits/a5800080b802b2604c72ee37cf9ccf1bbf5ce0c3.